testdata/pruner/task/taskrun-for-pruner.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
Can't we have just one run in this file and specify it multiple times in `auto-prune.spec`?

Reply:
The problem with having a single file is, it cannot be called twice as the task/pipeline as `oc create -f` fails since the task/pipeline already presents.